admin.php

```diff
@@ -0,0 +1,52 @@
+<?php
+require_once './database.php';
+
+//get projects
+$sql = "SELECT * FROM `project`;";
+$query = $db->prepare($sql);
```

Comment:
dont need to prepare and execute if you have no user data, can just use query

Reply:
will refactor this post-MVP